### PR TITLE
Week 4 Day 1: Best Practices for Installability 

### PR DESCRIPTION
**Pull request by zateutsch**
---

⚠️ Merge/Release date 3/7

---

### CODE REVIEW

File: docs/30DaysOfPWA/platforms-practices/01.md [hunk 1]
```diff
@@ -1,14 +1,106 @@
 # 4.1: Installability for PWAs
 
-## *Coming soon!*
+**Author: Zach Teutsch [@devteutsch](https://twitter.com/devteutsch)**
 
 ![Placeholder Banner Only. Replace when final assets ready.](_media/day1.png)
 
-### WHAT WE'LL COVER TODAY
+Welcome to Week 4, Day 1 of the [30 Days of PWA](https://aka.ms/learn-pwa/30Days-blog) series! Now that we've seen how to start building PWAs and what basic and advanced capabilites they bring, it's time to dive into what a _good_ PWA really looks like it, and how we can start to build quality applications ourselves. Week 4 will be all about some of the best practices and technologies you can use to really polish up your PWA.
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/platforms-practices/01.md:7]: Maybe it's just me but "start building PWAs" feels like we're at the beginning of the series, not the end.
Not sure how to rephrase it, but I'd want this intro to say something like we've gone through all the steps, from learning basic blocks, to coding, to debugging and testing, and finally packaging for stores. So now it's time to wrap things up with best practices. 
Does that make sense?

File: docs/30DaysOfPWA/platforms-practices/01.md [hunk 2]
```diff
@@ -1,14 +1,106 @@
...
+
+We'll learn more about best practices for PWA UI/UX on [Day 7](07.md) of Platforms and practices week.
+
+**Consistent offline behavior**
+
+Even though Twitter relies pretty heavily on connectivity, it still maintains as much of the app interaction as possible when connection is lost (just like any native app would).
+
+When connection is lost, the app still loads its view and the UI is still navigable. If there are any tweets stored locally, those will be displayed. If there aren't, the user will get a notice that doesn't interfere with the rest of the UI:
+
+![](_media/twitter-offline.png#test)
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/platforms-practices/01.md:62]: Missing alt text here.

File: docs/30DaysOfPWA/platforms-practices/01.md [hunk 3]
```diff
@@ -1,14 +1,106 @@
...
+
+We already covered how to use all these capabilities and more in [Week 2: Advanced Capabilities.](../advanced-capabilities) Go check them out!
+
+**Published to the Microsoft Store**
+
+Twitter takes advantage of one the best parts about PWAs: discoverability on the web _and_ in app stores.
+
+![Twitter in the Microsoft Store](_media/twitter-in-store.png)
+
+The same PWA the can be installed from the browser can be distributed in app stores. You can learn more about packaging for the stores in our [article on PWABuilder](../dev-tools/02.md) and more info about best practices for distribution to different app stores is coming [later this week.](03.md)
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/platforms-practices/01.md:84]: I think you probably want to link to day 7 of week 2, not day 2, right?

File: docs/30DaysOfPWA/platforms-practices/01.md [hunk 4]
```diff
@@ -1,14 +1,106 @@
...
+
+
+## Summary
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/platforms-practices/01.md:87]: Before the summary, what do you think about showing other great PWAs too. I don't think we should go in as much details as what you've done for Twitter (which is great, btw), but adding a couple of other telling examples might be good for people to get an even better understanding of what a great PWA is.
https://www.pwastats.com/ might provide a good list of inspiration for this. I can also think of Photopea, and Adobe Creative Cloud Express.

**Review by captainbrosset — APPROVED** [review]
Great, short, simple, and to the point. I like it!

File: docs/30DaysOfPWA/platforms-practices/01.md [hunk 5]
```diff
@@ -1,14 +1,106 @@
...
+**OS integration and plenty of web capabilities**
+
+Twitter also makes great use of available web capabilities to integrate with your device. Just to list some:
+
+* Push notifications
+* Sharing from Twitter
+* Sharing to Twitter
+* Badging
+* Shortcuts
+
```
Comment (jgw96) [inline docs/30DaysOfPWA/platforms-practices/01.md:75]: Maybe add a screenshot of the Twitter shortcuts or something? Just something to drive this point home some more.

File: docs/30DaysOfPWA/platforms-practices/01.md (hunk 4 above)
Comment (jgw96) [reply docs/30DaysOfPWA/platforms-practices/01.md:87]: Clipchamp is another good one

File: docs/30DaysOfPWA/platforms-practices/01.md [hunk 6]
```diff
@@ -1,14 +1,106 @@
...
+We already covered how to use all these capabilities and more in [Week 2: Advanced Capabilities.](../advanced-capabilities) Go check them out!
+
+**Published to the Microsoft Store**
+
+Twitter takes advantage of one the best parts about PWAs: discoverability on the web _and_ in app stores.
```
Comment (jgw96) [inline docs/30DaysOfPWA/platforms-practices/01.md:80]: Maybe change this to `discoverability and install-ability` just to point out that PWAs can be installed from the browser too.

File: docs/30DaysOfPWA/platforms-practices/01.md (hunk 2 above)
Comment (zateutsch) [reply docs/30DaysOfPWA/platforms-practices/01.md:62]:
```suggestion
![Twitter offline warning](_media/twitter-offline.png#test)
```

File: docs/30DaysOfPWA/platforms-practices/01.md (hunk 3 above)
Comment (zateutsch) [reply docs/30DaysOfPWA/platforms-practices/01.md:84]:
```suggestion
The same PWA the can be installed from the browser can be distributed in app stores. You can learn more about packaging for the stores in our [article on PWABuilder](../dev-tools/07.md) and more info about best practices for distribution to different app stores is coming [later this week.](03.md)
```

File: docs/30DaysOfPWA/platforms-practices/01.md (hunk 6 above)
Comment (zateutsch) [reply docs/30DaysOfPWA/platforms-practices/01.md:80]:
```suggestion
Twitter takes advantage of one the best parts about PWAs: discoverability and installability on the web _and_ in app stores.
```

File: docs/30DaysOfPWA/platforms-practices/01.md (hunk 4 above)
Comment (zateutsch) [reply docs/30DaysOfPWA/platforms-practices/01.md:87]: @captainbrosset @jgw96  My original plan was to show multiple PWAs (including clipchamp), however, I found that driving home a single PWA is helpful to showcase a top-to-bottom experience, as opposed to cherry picking features from different apps. 

It works well for pushing the idea that adding a web capability here and there doesn't necessarily make a good PWA.

File: docs/30DaysOfPWA/platforms-practices/01.md (hunk 5 above)
Comment (zateutsch) [reply docs/30DaysOfPWA/platforms-practices/01.md:75]: I like this idea.

File: docs/30DaysOfPWA/platforms-practices/01.md [hunk 7]
```diff
@@ -1,14 +1,108 @@
 # 4.1: Installability for PWAs
 
-## *Coming soon!*
+**Author: Zach Teutsch [@devteutsch](https://twitter.com/devteutsch)**
+
+Welcome to Week 4, Day 1 of the [30 Days of PWA](https://aka.ms/learn-pwa/30Days-blog) series! Now that we've seen how to start building PWAs and what basic and advanced capabilites they bring, it's time to dive into what a _good_ PWA really looks like it, and how we can turn our PWAs into quality application experiences. Week 4 is all about some of the best practices and technologies you can use to really polish up your PWA.
```
Comment (beth-panx) [inline docs/30DaysOfPWA/platforms-practices/01.md:5]:
```suggestion
Welcome to Week 4, Day 1 of the [30 Days of PWA](https://aka.ms/learn-pwa/30Days-blog) series! Now that we've seen how to [start building PWAs](../core-concepts/) and what basic and [advanced capabilities](../advanced-capabilities/) they bring, it's time to dive into what a _good_ PWA really looks like, and how we can turn our PWAs into quality application experiences. Week 4 is all about some of the best practices and technologies you can use to really polish up your PWA.
```

File: docs/30DaysOfPWA/platforms-practices/01.md [hunk 8]
```diff
@@ -1,14 +1,108 @@
...
+Think about some of your favorite web experiences that you use or have built yourself. 
+
+Would any of these experiences make good PWAs? Would any of them not? 
+
+What could these experiences gain by taking advantage of some of the capabilites that PWAs bring to the table?
+
+### **Resources**
+
+* **[Microsoft PWA Overview](https://aka.ms/learn-pwa/30days-4.1/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium)**
+* **[PWA Advanced Capabilities Week](../advanced-capabilities)**
```
Comment (beth-panx) [inline docs/30DaysOfPWA/platforms-practices/01.md:105]:
```suggestion
* **[PWA Advanced Capabilities Week](../advanced-capabilities/)**
```

For some reason the generated site won't redirect without the `/`.

**Review by beth-panx — APPROVED** [review]
Awesome job! 🚀

**Pull request merged**